atter-logic/src/commonMain/kotlin/com.attendance.checker/date/KDate.kt

```diff
@@ -0,0 +1,10 @@
+package date
```

Comment:
It seems like package does not match file location.
I'll try to introduce few automated checks to avoid codestyle issues pop up in the future.